.circleci/config.yml

```diff
@@ -89,6 +89,7 @@ jobs:
           name: Download source/check/compile
           working_directory: ~/build_dir
           command: |
+             set +o pipefail
```

Comment:
Actually pipefail makes package removal fail
https://github.com/openwrt/packages/pull/7454#issuecomment-445059224

Reply:
If this PR were to be rebased -, the line you highlighted will "fix" the strange exit failure. However, if the  "git diff", etc. finds a Makefile in the PR, it will proceed. We probably need a better check here.

Reply:
Indeed, I got confused by bash syntax, + to disable, - to enable